### ISSUE TITLE
Install scripts not running (abbreviated manifest missing hasInstallScript)

### ISSUE THREAD

**Issue by pasieronen** [verdaccio/verdaccio#3366]
**Your Environment**
  
 * **verdaccio version**: 5.15.3
 * **node version**: 16.17.0
 * **package manager**: npm@8.15.0
 * **os**: mac
 * **platform**: npm
 
**Describe the bug**

When installing a new package (not from package-lock.json) from verdaccio, install scripts are not run. It looks like
the abbreviated manifest introduced in 5.15.0 does not have correct value for "hasInstallScript" field.

**To Reproduce**

In an empty directory start:
```
npx verdaccio@5.15.3
```

In another empty directory, run:
```
NPM_CONFIG_REGISTRY=http://localhost:4873 npm i puppeteer@17.1.2 --foreground-scripts
grep hasInstallScript package-lock.json
```

Result with 5.15.3: no script output printed, no "hasInstallScripts" lines in package-lock.json.

**Expected behavior**

Result with 5.14.0: Puppeteer's install script is run (prints output like "Downloading Chromium..."), package-lock.json contains a line with `"hasInstallScript": true`

**Screenshots, server logs, package manager log**

When running 
```
curl -H 'Accept: application/vnd.npm.install-v1+json' http://localhost:4873/puppeteer 
````

the response does not have any "hasInstallScript" fields. When running the same against registry.npmjs.org, the response has "hasInstallScript" fields set to true.

**Configuration File (cat ~/.config/verdaccio/config.yaml)**

**Environment information**

**Debugging output**

**Contribute to Verdaccio**

- [ ] I'm willing to fix this bug 🥇

**Comment by ParryingDagger** [comment]
https://github.com/verdaccio/verdaccio/blob/master/packages/store/src/storage.ts#L552

**Comment by juanpicado** [comment]
solved at 5.18.0
